m4_sessions/1710-inning/group_one.md

```diff
@@ -1,10 +1,27 @@
-Group One
+# Group One
+## Session Title: Toxic Masculinity & Nerd Culture
+## Description: Why should students come to this session?
+* Nerd culture is now part of the zeitgeist. Come join us and 		learn why you got to get over your idea of what a nerd is, 		dude.
 
-Session Title: 
-Description: Why should students come to this session? 
+## Session Outcomes: What will students leave with or gain from this session?
+* Intro: What’s your favorite nerd thing?
+* Awareness of what nerd culture encompasses and why the term ‘nerd’ isn’t specific.
+* Awareness of how toxic masculinity effects media and why there’s this protectionism in ‘nerd’ media (aka. misogyny)
+* Examples in media of this actually in media
+* Real world examples of ways it can harm (Dr. Disrespect)
+* Discussion - Where do you see the effects in media and in your own life? How has this affected you? 
```

Comment:
great discussion question.  Maybe thinking about ways it has impacted you positively and negatively.